src/vs/workbench/services/sessionSync/browser/sessionSyncWorkbenchService.ts

```diff
@@ -198,22 +198,31 @@ export class SessionSyncWorkbenchService extends Disposable implements ISessionS
 	 * Returns all authentication sessions available from {@link getAuthenticationProviders}.
 	 */
 	private async getAllSessions() {
-		const options: ExistingSession[] = [];
 		const authenticationProviders = await this.getAuthenticationProviders();
+		const accounts = new Map<string, ExistingSession>();
+		let currentSession: ExistingSession | undefined;
 
 		for (const provider of authenticationProviders) {
 			const sessions = await this.authenticationService.getSessions(provider.id, provider.scopes);
 
 			for (const session of sessions) {
-				options.push({
+				const item = {
 					label: session.account.label,
```

Comment:
Not guaranteed to be unique. You should use `session.account.id` as the key of your Map.

Reply:
Yep that should be fixed. cc @sandy081